### ISSUE TITLE
Error maven install 15.0.6

### ISSUE THREAD

**Issue by Tz1Git** [vaadin/flow#8275]
I didn't install 15.0.5 because it was a compilation error, now it came out at 15.0.6 and tried again. The error is now another:

Run As -> Maven Install 

Maven in production...

```
[INFO] Running `npm install` to resolve and optionally download frontend dependencies. This may take a moment, please stand by...

C:\Program Files\nodejs\npm.cmd \ 
    --no-update-notifier --no-audit install
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm ERR! Unexpected end of JSON input while parsing near '...erlay/-/vaadin-overla'

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\PC1\AppData\Roaming\npm-cache\_logs\2020-05-07T20_45_11_577Z-debug.log
```

Log:

0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   '--no-update-notifier',
1 verbose cli   '--no-audit',
1 verbose cli   'install'
1 verbose cli ]

Final file log:

1793 verbose stack SyntaxError: Unexpected end of JSON input while parsing near '...erlay/-/vaadin-overla'
1793 verbose stack     at JSON.parse (<anonymous>)
1793 verbose stack     at parseJson (C:\Program Files\nodejs\node_modules\npm\node_modules\json-parse-better-errors\index.js:7:17)
1793 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-fetch-npm\src\body.js:96:50
1793 verbose stack     at processTicksAndRejections (internal/process/task_queues.js:85:5)
1794 verbose cwd C:\Users\PC1\git\ProAtivo
1795 verbose Windows_NT 10.0.18363
1796 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "--no-update-notifier" "--no-audit" "install"
1797 verbose node v12.8.1
1798 verbose npm  v6.10.2
1799 error Unexpected end of JSON input while parsing near '...erlay/-/vaadin-overla'
1800 verbose exit [ 1, true ]

**Comment by Tz1Git** [comment]
I managed to overcome the problem:

I completely removed the project;
I reimported again to the eclipse
I put the option

```
<configuration>
<pnpmEnable> true </pnpmEnable>
</configuration>

```
in pom.xml and ran it again. It worked. But even so, I tried it, I can't say exactly why it worked.

**Comment by haijian-vaadin** [comment]
Hi @Tz1Git, we just tried to reproduce the problem with our [spring starter](https://github.com/vaadin/skeleton-starter-flow-spring), it works fine there. 

Could you help to share with us a project that we can reproduce the mentioned problem?

**Comment by Tz1Git** [comment]
Hello, I don't use spring boot only CDI Java EE.

As I needed to solve this because it didn't even go back to 15.0.4, I no longer have access to the error, because as I said, I removed the entire project, switched to PNPM and then ran the system from scratch. It worked, then I went back to my original configuration and now it's ok, but I had to change the whole pom.xml to its version on the start page.

So it seems to me that it is not just about changing the version number, because strange things have been happening since version 15.0.5 when I do this. We'll see from now on.

But I saw that someone else also reported, maybe she can send the project with the error: [(https://github.com/vaadin/flow/issues/8278)]
